lib/ansible/modules/cloud/vmware/vmware_cluster_ha.py

```diff
@@ -65,19 +66,66 @@
       type: str
       choices: ['vmAndAppMonitoring', 'vmMonitoringOnly', 'vmMonitoringDisabled']
       default: 'vmMonitoringDisabled'
-    ha_failover_level:
```

Comment:
Add a note stating this parameter is deprecated (replaced by `slot_based_admission_control` ) and will be removed in 2.13

Reply:
> Add a note stating this parameter is deprecated (replaced by `slot_based_admission_control` ) and will be removed in 2.13

Are you sure? This module doesn't exist in 2.8. It looks a bit weird to me if a **new** module already has deprecated parameters... after all, I can't break any existing playbooks. At least no playbooks that use a stable version of ansible. The change between 2.8 and 2.9 ist not the deprecation of this parameter, the change is the new module.

Reply:
Ah ! sorry my bad. I didn't realise this module is new module.